espnet2/tasks/abs_task.py

```diff
@@ -846,6 +846,55 @@ class ArgumentDefaultsRawTextHelpFormatter(
             "e.g., 'bias_weight_decay': False will set zero weight decay for bias "
             "params. See also espnet2.optimizers.optim_groups.configure_optimizer.",
         )
+        group.add_argument(
```

Comment:
The following arguments are task-specific. You don't have to put biasing-related arguments here. Instead, you can put them in the configuration `model_conf`. They will be passed  to the `espnet_model`.

Reply:
So actually there are biasing-related arguments that I need to pass to the joint network as well. Is it better to keep these arguments, or is it better to have those in both model_conf and joint_net_conf in the config file?